zppy/ilamb.py

```diff
@@ -31,8 +31,10 @@ def ilamb(config: ConfigObj, script_dir: str, existing_bundles, job_ids_file):
         return existing_bundles
 
     # --- Generate and submit ilamb scripts ---
-    dependencies: List[str] = []
     for c in tasks:
+
+        dependencies: List[str] = []
```

Comment:
All tasks that weren't changed in this PR already have `dependencies` reset inside the `for c in tasks`